examples/01-getting-started.ipynb

```diff
@@ -0,0 +1,257 @@
+{
```

Comment:
<p>Whats this showing? How does this list of numbers thats return from <code>model.evaluate</code> let users interpret the performance of their model  ? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
I added the explanation. However, we should add a FEA request to provide a dictionary instead of a list